scripts/zones/Garlaige_Citadel_[S]/npcs/Randecque.lua

```diff
@@ -0,0 +1,55 @@
+-----------------------------------
+-- Area: Garlaige Citadel [S]
+--  NPC:Randecque
+-- @pos 61 -6 137 164
+-- Notes: Gives Red Letter required to start "Steamed Rams"
+-----------------------------------
+package.loaded["scripts/zones/Garlaige_Citadel_[S]/TextIDs"] = nil;
+-----------------------------------
+
+require("scripts/globals/settings");
+require("scripts/globals/keyitems");
+require("scripts/zones/Garlaige_Citadel_[S]/TextIDs");
+
+-----------------------------------
+-- onTrade Action
+-----------------------------------
+
+function onTrade(player,npc,trade)
+end;
+
+-----------------------------------
+-- onTrigger Action
+-----------------------------------
+
+function onTrigger(player,npc)
+    if (player:getVar("Used_RED_RECOMMENDATION_LETTER") == 1) then

```

Comment:
Nope. "seen" trips if you enter menu and click it, not if its been used. And I could just check quest status either, as its possible to do the quest without the KI at all. I hate dangling vars though (player variables that get set and left forever, never being zeroed out) so I am looking to change this.

..I think perhaps checking campaign allegiance along with having/not having the KI here would be enough. I don't think you can get any more letters once you've joined a nation, but I have to check on that in retail 1st. I don't trust wiki on it. I know you can get all 3 beforehand though and use them each after the fact way later.